ui-tests/cypress/support/pageObjects/users/profileTypes/EditPage.js

```diff
@@ -0,0 +1,44 @@
+import { htmlElements } from '../../WebElement';
+
+import Content from '../../app/Content.js';
+
+import AppPage from '../../app/AppPage.js';
+
+import ProfileTypesPage from './ProfileTypesPage';
+
+export default class AddPage extends Content {
```

Comment:
This class should be named 'EditPage' and if it has the same structure as 'AddPage' there's no need to declare to separate classes

Reply:
It's intentionally separated since the 'EditPage' has some additional form sections that aren't included yet since these changes cover only the basic tests.

Reply:
just rename it then 😉

Reply:
Oh, I honestly didn't notice that 😄 Will do that.